scripts/periodic/410.pkg-audit.in

```diff
@@ -156,6 +157,19 @@ audit_pkgs_all() {
 	esac
 
 	for j in $jails ; do
+		# ignore some jails
+		if [ -n "$security_status_pkgaudit_jails_ignore" ]; then
+			# we iterate to get exact matches because we want substring matches
+			# foo should not match foo.bar
+			for ignore in $security_status_pkgaudit_jails_ignore ; do
+				if [ "${j%|*}" == "$ignore" ]; then
```

Comment:
== is a bashism (even if it works with ou sh(1) since recently)

Reply:
We can't have that crap in there. ;). I'll fix it, test again, etc.